<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Автомастерская</string>
  </property>
  <widget class="QLabel" name="label_op1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>153</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op0">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>125</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>181</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_clients_title">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>59</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Клиенты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op_title">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>106</y>
     <width>138</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Отдел ремонта:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_model">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>старт</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_op1_m">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>154</y>
     <width>125</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op1_pm">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>154</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_op1_d">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>154</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op2_pm">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>182</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_op2_d">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>182</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_op2_m">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>182</y>
     <width>125</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_client_pm">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>79</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_client_m">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>79</y>
     <width>125</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_client_d">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>79</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_op0_m">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>126</y>
     <width>125</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>45</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op0_pm">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>126</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_op0_d">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>126</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_client_count_title">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>18</y>
     <width>115</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Всего клиентов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_client_count">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op0_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>239</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_car_m">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>240</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op_title_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>220</y>
     <width>138</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Автомойка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op_title_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>138</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Отдел покраски:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op0_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>289</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uzi_m">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>290</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter0_p">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op_title_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вероятность использования &lt;br/&gt;услуги мойки и покраски&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter2_p">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter1_p">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_car_d">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op2_pm_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uzi_d">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op2_pm_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op_title_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вероятность покраски&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_car0_p">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op_title_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Макс. длина очереди:&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_op_title_7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Количество машин,&lt;br/&gt;прошедших через отдел:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter0_max">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter1_max">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter2_max">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_car_max">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uzi_max">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_car_all">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter1_all">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter2_all">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_uzi_all">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ter0_all">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
